kannada/romaanchanavee_kannada/romaanchanavee_kannada.md

```diff
@@ -22,10 +22,12 @@
   ೧.  ಹಳಗನ್ನಡ ಕ್ರಿ.ಶ. ೪೫೦ರಿಂದ ಕ್ರಿ.ಶ. ೧೨೦೦ರವರೆಗೆ,
   ೨.  ನಡುಗನ್ನಡ ಕ್ರಿ. ಶ. ೧೨೦೦ರಿಂದ ಕ್ರಿ.ಶ. ೧೭೦೦ರವರೆಗೆ ಮತ್ತು
   ೩.  ಹೊಸಗನ್ನಡ ಕ್ರಿ. ಶ. ೧೭೦೦ರಿಂದ ಪ್ರಸ್ತುತ ಕಾಲಘಟ್ಟದವರೆಗೆ.
- 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.